index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <header >
+        <div class="container-main first-nav">
+                <ul class="white">
+                    
+                    <li><a href="#"><img src="./assets/mobile/facebook.png" alt="fb-logo" width="10px" height="10px"></a></li>
+                    <li><a href="#"><img src="./assets/mobile/twitter.png " alt="twit-log" width="10px" height="10px"></a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Log Out</a></li>
+                </ul>
+        </div>
+        <nav>
+            <div class="container-main">
+                <div class="container-start second-nav">
+                    <a href="#"><label for="logo"><img class="logo" src="./assets/mobile/logo1.png" alt="logo"</label></a>
+                    <ul>
+                        <li><a class="active" href="#">Home</a></li>
+                        <li><a class="active" href="./about.html">About</a></li>
+                        <li><a class="active" href="#courses">Courses</a></li>
+                        <li><a class="active" href="#teachers">Teachers</a></li>
+                        <li><a class="active" href="#partners">Partners</a></li>
+                        <li class="bb-btn">
+                            <button class="cc-btn">Al-Noor Campaign</button>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+        <div class="mobile-option mobile-option2 ">
+            <div class="head-dom">
+              <button class="exit">
+                &#x2716;
+              </button>
+            </div>
+          <ul class="mob-style">
+            <li><a class="nav-link" href="#">Home</a></li>
+            <li><a class="nav-link" href="./about.html">About</a></li>
+            <li><a class="nav-link" href="#holyquran">Holy Quran</a></li>
+            <li><a class="nav-link" href="#courses">Courses</a></li>
+            <li><a class="nav-link" href="#teachers">Teachers</a></li>
+            <li><a class="nav-link" href="#partners">Partners</a></li>
+        </ul>
```

Comment:
- [ ] I cannot find this section in the given design. Please recheck and remove it.

Reply:
Seems you don't check the mobile version... I build a different nav section for the hamburger menu. And I will not remove this because its a project requirement! Thanks.